mathenger-api/src/main/java/com/example/mathengerapi/integrations/dspace/DSpaceMessageHandler.java

```diff
@@ -30,6 +35,17 @@ private void handleBotMessage(MessageSent event) {
         var message = event.getText();
         if ("/community".equals(message)) {
             botCommandsHandler.handleAllCommunities(event.getChatId());
+        } else if (message.matches("/community_" + UUID_REGEX)) {
+            UUID communityId = UUID.fromString(message.substring(11, message.length()));
+            botCommandsHandler.handleAllCollectionsOfCommunity(event.getChatId(), communityId);
+        } else if (message.matches("/colpublications_" + UUID_REGEX)) {
+            UUID collectionId = UUID.fromString(message.substring(17, message.length()));
```

Comment:
It would be nice to replace these magic numbers (11, 17) with some kind of general logic but I guess it can be done in scope of separate ticket